metrics/metrics.go

```diff
@@ -53,6 +53,10 @@ var (
 	ProtocolID, _ = tag.NewKey("proto")
 	Direction, _  = tag.NewKey("direction")
 	UseFD, _      = tag.NewKey("use_fd")
+
+	// Subnet ID
+	SubnetID, _    = tag.NewKey("subnet_id")
+	SubnetLevel, _ = tag.NewKey("subnet_level") // The subnet level, the number of parents it has
```

Comment:
Number of ancestors? Subnets can only have one parent

Reply:
Number of ancestors seems to be the more correct term